test/helpers/write-comparor.js

```diff
@@ -82,15 +79,21 @@ function postscribeResults(...def) {
       postscribe(el, d, opts);
     }
 
-    return dfd.promise;
+    return dfd.promise();
   })).then(() => el.innerHTML);
 }
 
 function transform(result, fns) {
-  return _.reduce(fns, (acc, f) => f(acc), result);
+  let transformed = result;
+  $.map(fns, f => {
+    transformed = f(transformed)
+  });
+
+  return transformed;
 }
 
-function removeScripts(node) {
-  _.forEach(_.toArray(node.getElementsByTagName('script')), n => n.parentNode.removeChild(n));
+function removeHiddenElements(node) {
+  $('script', node).remove();
+  $('style', node).remove();
```

Comment:
They write to different positions. I'm thinking I should remove them, but then compare which were written for greater accuracy.